cumulus_library/study_parser.py

```diff
@@ -307,18 +373,85 @@ def run_table_builder(
         for file in self.get_table_builder_file_list():
             self._load_and_execute_builder(file, cursor, schema, verbose)
 
-    def run_counts_builder(
+    def run_counts_builders(
         self, cursor: DatabaseCursor, schema: str, verbose: bool = False
     ) -> None:
         """Loads counts modules from a manifest and executes code via BaseTableBuilder
 
+        While a count is a form of statistics, it is treated separately from other
+        statistics because it is, by design, always going to be static against a
+        given dataset, where other statistical methods may use sampling techniques
+        or adjustable input parameters that may need to be preserved for later review.
+
         :param cursor: A PEP-249 compatible cursor object
         :param schema: The name of the schema to write tables to
         :param verbose: toggle from progress bar to query output
         """
         for file in self.get_counts_builder_file_list():
             self._load_and_execute_builder(file, cursor, schema, verbose)
 
+    def run_statistics_builders(
+        self,
+        cursor: DatabaseCursor,
+        schema: str,
+        verbose: bool = False,
+        stats_build: bool = False,
+    ) -> None:
+        """Loads statistics modules from toml definitions and executes
+
+        :param cursor: A PEP-249 compatible cursor object
+        :param schema: The name of the schema to write tables to
+        :keyword verbose: toggle from progress bar to query output
+        :keyword stats_build: If true, will run statistical sampling & table generation
+        """
+        if not stats_build:
+            return
```

Comment:
nit: this is an odd method construction. I understand why you might end up here reasonably, I'm just going to prod you to see if there's a better way to get at the flow you want. If not, this is fine.

It especially triggered me because `stats_build=False` is the default.

Reply:
Here's what i believe stats generation lifecycle to look like:
- stats, if they exist in a study, are always run if they haven't been run before
- after this, stats should :not: be run
- a researcher may say 'ok, i've changed [x,y,z] and now i'd like to run a new sampling experiment, let me explicitly invoke it'
So some sort of 'usually false' workflow belong someplace. It might be worth nattering a bit about the the right layer of seperation between the Builder and the Parser (and should the Builder be named something different)? but it could move to the builder.